docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -29,7 +29,7 @@ The easiest way to install the Azure CLI is through a script maintained by the A
 If you wish to inspect the contents of the script yourself before executing, download the script first using `curl` and inspect it in your favorite text editor.
 
 ```bash
-curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+curl -fsSL https://aka.ms/InstallAzureCLIDeb | sudo bash
```

Comment:
I am not sure about the potential impact of adding [`-f`](https://curl.se/docs/manpage.html#-f) and [`-S`](https://curl.se/docs/manpage.html#-S). It would be safer not to change it.

Reply:
"be silent, unless it fails"
but I'll revert it.